src/server/qtquick/private/wquickxdgdecorationmanager.cpp

```diff
@@ -1,8 +1,13 @@
 // Copyright (C) 2023 rewine <luhongxu@deepin.org>.
 // SPDX-License-Identifier: Apache-2.0 OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
 
+#include "qwxdgshell.h"
```

Comment:
```suggestion
#include <qwxdgshell.h>
```
and move behind waylib headers